app/_posts/2024-11-05-kuma-2-9-0.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Kuma 2.9 release with MeshTLS, Service redesign, GUI updates and more...
+description: Kuma 2.9 release with MeshTLS, Service redesign, GUI updates and more...
+date: 2024-11-05
+headerImage: /assets/images/blog/test-header.jpg
+tags:
+  - Release
+---
+
+We're excited to announce the biggest release of Kuma since 2.0! 
+In this release we offer a complete beta release of our service rework and namespaced policies.
+
+# Notable features
+In Kuma 2.9.x we've focused on 3 main areas:
+* `MeshTLS` which provides a more granular way to roll out strict Mutual TLS.
+* New resources `MeshService` and `MeshMultiZoneService` which provide a more intuitive and powerful way to manage services in your mesh.
+* A complete rework of our destination policy attachment which works with the new services.
+* Introduction of producer and consumer policies, enabling more flexibility for Kubernetes users.
+
+This set of features greatly simplifies Kuma's user experience, by making it clearer how services exist and by attaching policies
+to specific resources. These changes are also fully integrated with our GUI thus making things incredible easy to understand and troubleshoot.
+
+This release marks an important step in simplifying the Kuma user experience, empowering both key user roles: mesh administrators and service owners.
+
+Feel free to check our [release notes](https://github.com/kumahq/kuma/releases/tag/2.9.0) for the full list of changes.
+
+## New TLS policy
+
+Mutual TLS is one of the most common use cases in Kuma. This is because it's the base on which you can build fully secured  trust environments.
+The most common way to roll out mutual TLS is by first running services in **permissive mode,** in this mode, while the mesh will verify client certificates,
+it will still allow unsecured connection.
+You can then progressively remove unsecure connections by moving clients inside the mesh and once it's complete switch to **strict mode.**
```

Comment:
```suggestion
You can then progressively remove insecure connections by moving clients inside the mesh and then finally switch to **strict mode.**
```

Reply:
So I looked into this one and clearly I think unsecure is more correct than insecure

The def on unsecure is:

unsecure meaning https://g.co/kgs/noU1tdd